lib/elastic_apm/transport/serializers/span_serializer.rb

```diff
@@ -78,7 +78,10 @@ def build_destination(destination)
                 resource: keyword_field(destination.resource),
                 type: keyword_field(destination.type)
               }
-            }
+            }.tap do |dest|
+              dest[:address] = keyword_field(destination.address)
+              dest[:port] = destination.port
+            end
```

Comment:
Why not just use regular `key: value` here instead of the `tap`?

Reply:
because that would make too much sense!